cmd/manager/main.go

```diff
@@ -1020,13 +1026,15 @@ func setupWebhook(
 	// wait for the secret to be populated in the local filesystem before returning
 	interval := time.Second * 1
 	timeout := time.Second * 30
-	keyPath := filepath.Join(mgr.GetWebhookServer().CertDir, certificates.CertFileName)
+	keyPath := filepath.Join(webhookCertDir, certificates.CertFileName)
 	log.Info("Polling for the webhook certificate to be available", "path", keyPath)
-	err := wait.PollImmediate(interval, timeout, func() (bool, error) {
+	//nolint:staticcheck
+	err := wait.PollImmediateWithContext(ctx, interval, timeout, func(_ context.Context) (bool, error) {
```

Comment:
`PollImmediate` is deprecated but there was a mistake and the new functions are also deprecated, hence the `nolint:staticcheck` to prevent the go linter from complaining.

- https://github.com/kubernetes/apimachinery/issues/153